src/notation/internal/notationplaybackmock.cpp

```diff
@@ -0,0 +1,175 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "notationplaybackmock.h"
+
+using namespace mu;
+using namespace mu::notation;
+using namespace mu::engraving;
+using namespace muse;
+using namespace muse::midi;
+using namespace muse::async;
+
+InstrumentTrackId NotationPlaybackMock::m_trackId;
+muse::mpe::PlaybackData NotationPlaybackMock::m_playbackData;
+LoopBoundaries NotationPlaybackMock::m_loopBoundaries;
+Tempo NotationPlaybackMock::m_tempo;
+
+NotationPlaybackMock::NotationPlaybackMock()
+{
+}
+
+void NotationPlaybackMock::init()
+{
+}
+
+const engraving::InstrumentTrackId& NotationPlaybackMock::metronomeTrackId() const
+{
+    return m_trackId;
```

Comment:
Usually, we do it differently -- directly declare static dummy data inside the function body:

```
const engraving::InstrumentTrackId& NotationPlaybackMock::metronomeTrackId() const
{
    static const engraving::InstrumentTrackId dummy;
    return dummy;
}

```